src/Resources/config/mappings/ShopUser.yaml

```diff
@@ -22,11 +22,6 @@ Sylius\Component\Core\Model\ShopUser:
             faker: dateTime
         verifiedAt:
             faker: dateTime
-        locked:
-            faker: boolean
-            args: [100]
-        credentialsExpireAt:
-            faker: dateTime
```

Comment:
i don't get why you deleted those lines ?

Reply:
This fields is not available on Sylius 2.0